application-load-balancer/tests/unit.tftest.hcl

```diff
@@ -1,3 +1,5 @@
+mock_provider "aws" {}
+
 mock_provider "aws" {
   alias = "sandbox"
 }
```

Comment:
@chiaramapellimt If you question in Slack relates to the two providers above and the one below. 

I think the first one covers where we would be using the default providers and the others relate to specific accounts. E.g. https://github.com/uktrade/terraform-platform-modules/blob/main/application-load-balancer/main.tf#L110

Reply:
oooo I see